aafrooze/demo
# Stub out code for panacea project

The panacea project will cure all of the problems in the world, including
famine, disease, war, and the collected works of Stephanie Meyer. This commit
adds the initial stub code needed to run the project and inform the user of
success. Remaining implementation work needed to actually cure any problems.

Test Plan:
1. Run the code
2. Verify that output states all problems have been cured
3. Turn TV to CNN and confirm channel is no longer broadcasting as all problems
have been cured.
